scripts/qa_crowbarsetup.sh

```diff
@@ -1298,48 +1303,111 @@ function create_repos_yml_for_platform()
     for repo in $*; do
         repo_name=${repo%%=*}
         repo_url=${repo##*=}
-        if [ -d "$tftpboot_dir/$repo_name" ]; then
-            if [ -z "$platform_created" ]; then
-                echo "$platform:"
-                echo "  $arch:"
-                platform_created=1
-            fi
-
-            echo "    $repo_name:"
-            echo "      url: '$repo_url'"
+        if [ -z "$platform_created" ]; then
+            echo "$platform:"
+            echo "  $arch:"
+            platform_created=1
         fi
+
+        echo "    $repo_name:"
+        echo "      url: '$repo_url'"
     done
 }
 
 function create_repos_yml()
 {
     local repos_yml="/etc/crowbar/repos.yml"
     local tmp_yml=$(mktemp).yml
+    local additional_repos=
 
     echo --- > $tmp_yml
 
     if iscloudver 6; then
+        additional_repos=
+        if [ -n "$want_test_updates" -a "$want_test_updates" != "0" ] ; then
+            additional_repos+=" SLES12-SP1-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP1:/x86_64/update/"
+            additional_repos+=" SUSE-OpenStack-Cloud-6-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/OpenStack-Cloud:/6:/x86_64/update/"
+            [ $hacloud == 1 ] && additional_repos+=" SLE12-SP1-HA-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP1:/x86_64/update/"
+            [ -n "$deployceph" ] && additional_repos+=" SUSE-Enterprise-Storage-2.1-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/Storage:/2.1:/x86_64/update/"
+        fi
+        for devel_repo in ${want_devel_repos//,/ }; do
+            case "$devel_repo" in
+                ha)
+                    ;;
```

Comment:
There's no devel repo for HA yet, but HA team might consider having one, adding a TODO for this.